pysmt/test/test_bdd.py

```diff
@@ -115,6 +122,41 @@ def test_quantifier_eliminator(self):
         g = qelim(f, solver_name="bdd")
         self.assertEqual(g, self.y)
 
+    @skipIfSolverNotAvailable("bdd")
+    def test_reordering(self):
+        from pysmt.solvers.bdd import BddSolver
```

Comment:
We should extend the factory to be able to pass user_options to the solver. Thus this would simply be a call to the factory *Solver* .